src/nvim/ex_cmds.c

```diff
@@ -5535,6 +5535,9 @@ static buf_T *show_sub(exarg_T *eap, pos_T old_cusr,
 
   // We keep a special-purpose buffer around, but don't assume it exists.
   buf_T *preview_buf = bufnr ? buflist_findnr(bufnr) : 0;
+  // disable :leftabove/botright modifiers
+  // (especially ones that conflict with our win_split() call below)
+  cmdmod.split = 0;
   cmdmod.tab = 0;                 // disable :tab modifier
   cmdmod.noswapfile = true;       // disable swap for preview buffer
```

Comment:
Nit: would be nice to use the same inline, aligned comment form as these lines.

Reply:
Sure, done :+1: